modules/nf-core/hmtnote/annotate/environment.yml

```diff
@@ -2,4 +2,7 @@ channels:
   - conda-forge
   - bioconda
 dependencies:
-  - bioconda::hmtnote=0.7.2
+  - conda-forge::python=3.6 # Locked with the container
+  - conda-forge::pip
```

Comment:
You will need to fix the pip version as well

Reply:
Fixed to 21.3.1 that passes, thanks